message/producer.go

```diff
@@ -35,7 +29,7 @@ func (p InstanceCompletedProducer) Completed(e event.InstanceCompleted) error {
 	if avroError != nil {
 		return errors.Wrap(avroError, fmt.Sprintf("Marshaller.Marshal returned an error: event=%v", e))
 	}
-	p.Producer.Output() <- bytes
-	loggerP.Info("completed successfully", log.Data{"event": e})
+	p.Producer.Channels().Output <- bytes
+	log.Event(context.Background(), "completed successfully", log.INFO, log.Data{"event": e, "package": "message.InstanceCompletedProducer"})
```

Comment:
You will probably want to pass context into Completed function to be used in log line